backend/tests/v5/unit/processors/users.test.js

```diff
@@ -165,9 +175,85 @@ const testGenerateResetPasswordToken = () => {
 	});
 };
 
+const testSignUp = () => {
+	describe('Sign up a user', () => {
+		const newUserData = {
+			username: 'newUsername',
+			email: 'newEmail',
+			password: 'newPassword',
+			firstName: 'newname',
+		};
+
+		test('should sign a user up', async () => {
+			const sendVerifyUserEmailMock = Mailer.sendVerifyUserEmail.mockImplementation(() => {});
+
+			await Users.signUp(newUserData);
+			expect(addUserMock.mock.calls.length).toBe(1);
+			expect(addUserMock.mock.calls[0][0]).toEqual({ ...newUserData, token: exampleHashString });
+			expect(sendVerifyUserEmailMock.mock.calls.length).toBe(1);
+			expect(sendVerifyUserEmailMock.mock.calls[0][0]).toEqual(newUserData.email);
+			expect(sendVerifyUserEmailMock.mock.calls[0][1]).toEqual({
+				token: exampleHashString,
+				email: newUserData.email,
+				firstName: newUserData.firstName,
+				username: newUserData.username,
+			});
+		});
+
+		test('should sign a user up even if verification email fails to be sent', async () => {
+			const sendVerifyUserEmailMock = Mailer.sendVerifyUserEmail.mockImplementation(() => {
```

Comment:
Use Once function for anything you want to happen once so you don't screw over any additional tests in the future.
```suggestion
			Mailer.sendVerifyUserEmail.mockImplementationOnce(() => {
```